data.sql

```diff
@@ -8,6 +8,13 @@ INSERT INTO
     )
 values
     ('Agumon', '03-02-2020', 1, true, 10.23),
-    ('Gabumon', '2018-11-15', 2, false, 8),
-    ('Pikachu', '2021-01-15', 0, false, 6),
-    ('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+    ('Gabumon', '2018-11-15', 2, false, 8.00),
+    ('Pikachu', '2021-01-15', 0, false, 6.00),
+    ('Devimon', '2017-05-12', 5, true, 11.00),
+    ('Charmander', '2020-02-08', 0, false, 11.00),
+    ('Plantmon', '2021-11-15', 2, true, 5.70),
+    ('Squirtle', '1993-04-02', 3, false, 12.13),
+    ('Angemon', '2005-01-12', 1, true, 45.00),
```

Comment:
- [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), these weights have negative values, please kindly correct that.